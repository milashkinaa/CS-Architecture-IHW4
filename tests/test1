random numbers was generated
number of readers: 7
number of writers: 8

initial database: 
1 2 3 4 5 6 7 8 9 10 11 

writer 1 started his work with db
writer 1 changes 7th element in db from 7 to 3
database was also sorted
writer 1 finished his work with db
reader 6 started his work with db
reader 6 read 2th element. it's2
reader 7 started his work with db
reader 6 finished his work with db
reader 7 read 2th element. it's2
reader 7 finished his work with db
writer 4 started his work with db
writer 4 changes 6th element in db from 5 to 7
database was also sorted
writer 4 finished his work with db
writer 6 started his work with db
writer 6 changes 4th element in db from 3 to 3
database was also sorted
writer 6 finished his work with db
writer 8 started his work with db
writer 8 changes 8th element in db from 8 to 6
database was also sorted
writer 8 finished his work with db
reader 2 started his work with db
reader 1 started his work with db
reader 5 started his work with db
reader 5 read 1th element. it's1
reader 5 finished his work with db
reader 1 read 4th element. it's3
reader 1 finished his work with db
reader 2 read 11th element. it's11
reader 4 started his work with db
reader 3 started his work with db
reader 2 finished his work with db
reader 4 read 1th element. it's1
reader 4 finished his work with db
reader 3 read 1th element. it's1
reader 3 finished his work with db
writer 3 started his work with db
writer 3 changes 9th element in db from 9 to 13
database was also sorted
writer 3 finished his work with db
writer 2 started his work with db
writer 2 changes 6th element in db from 6 to 17
database was also sorted
writer 2 finished his work with db
writer 7 started his work with db
writer 7 changes 11th element in db from 17 to 7
database was also sorted
writer 7 finished his work with db
writer 5 started his work with db
writer 5 changes 1th element in db from 1 to 3
database was also sorted
writer 5 finished his work with db


new database: 
2 3 3 3 4 6 7 7 10 11 13 
