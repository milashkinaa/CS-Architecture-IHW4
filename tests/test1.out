1 reader has read recording with number 41
1 author changed the recording with number 41
0 author changed the recording with number 41

1 reader has read recording with number 67
1 author changed the recording with number 34
1 reader has read recording with number 34
0 author changed the recording with number 34
1 reader has read recording with number 0
1 author changed the recording with number 69
1 reader has read recording with number 69
0 author changed the recording with number 69
1 reader has read recording with number 24
1 author changed the recording with number 78
1 reader has read recording with number 78
0 author changed the recording with number 78
1 reader has read recording with number 58
1 author changed the recording with number 62
1 reader has read recording with number 62
0 author changed the recording with number 62
1 reader has read recording with number 64
1 author changed the recording with number 5
1 reader has read recording with number 5
0 author changed the recording with number 5
1 reader has read recording with number 45
1 author changed the recording with number 81
1 reader has read recording with number 81
0 author changed the recording with number 81
1 reader has read recording with number 27
1 author changed the recording with number 61
1 reader has read recording with number 61
0 author changed the recording with number 61
1 reader has read recording with number 91
1 author changed the recording with number 95
1 reader has read recording with number 95
0 author changed the recording with number 95
1 reader has read recording with number 42
1 author changed the recording with number 27
1 reader has read recording with number 27
0 author changed the recording with number 27
1 reader has read recording with number 36
1 author changed the recording with number 91
1 reader has read recording with number 91
0 author changed the recording with number 91
1 reader has read recording with number 4
1 author changed the recording with number 2
1 reader has read recording with number 2
0 author changed the recording with number 2
1 reader has read recording with number 53
1 author changed the recording with number 92
1 reader has read recording with number 92
0 author changed the recording with number 92
1 reader has read recording with number 82
1 author changed the recording with number 21
1 reader has read recording with number 21
0 author changed the recording with number 21
1 reader has read recording with number 16
1 author changed the recording with number 18
1 reader has read recording with number 18
0 author changed the recording with number 18
1 reader has read recording with number 95
1 author changed the recording with number 47
1 reader has read recording with number 47
0 author changed the recording with number 47
1 reader has read recording with number 26
1 author changed the recording with number 71
1 reader has read recording with number 71
0 author changed the recording with number 71
1 reader has read recording with number 38
1 author changed the recording with number 69
1 reader has read recording with number 69
0 author changed the recording with number 69
1 reader has read recording with number 12
1 author changed the recording with number 67
1 reader has read recording with number 67
0 author changed the recording with number 67
1 reader has read recording with number 99
1 author changed the recording with number 35
1 reader has read recording with number 35
0 author changed the recording with number 35
1 reader has read recording with number 94
1 author changed the recording with number 3
1 reader has read recording with number 3
0 author changed the recording with number 3
1 reader has read recording with number 11
1 author changed the recording with number 22
1 reader has read recording with number 22
0 author changed the recording with number 22
1 reader has read recording with number 33
1 author changed the recording with number 73
1 reader has read recording with number 73
0 author changed the recording with number 73
1 reader has read recording with number 64
1 author changed the recording with number 41
1 reader has read recording with number 41
0 author changed the recording with number 41
1 reader has read recording with number 11
1 author changed the recording with number 53
1 reader has read recording with number 53
0 author changed the recording with number 53
1 reader has read recording with number 68
Readers has read 50 recordings. Writers has changed 50 recordings.
